### DISCUSSION TITLE
Projections - Set properties for an Object field

### DISCUSSION THREAD

**Discussion by SalvadorCali** [mia-platform/community#27] [Q&A] [answered]
Hi everyone!
We have the necessity to configure the fields of a Projection that are Objects, and possibly set for some of their sub-fields specific cast functions. Is there a way to configure these options in the Projections section of the console, in a similar way to  the JSON schema field that appears in the MongoDB CRUD modal?

![image](https://user-images.githubusercontent.com/37811007/189944255-029f7061-c24d-4f14-9a00-35836c6d4989.png)

**Comment by flower-of-the-bridges** [comment, marked as answer]
Hi!

At the moment the only way to ensure validation for a field of type `Object` inside a Project is to define a dedicated cast function of type `Object`
